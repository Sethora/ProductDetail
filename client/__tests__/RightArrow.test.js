import React from 'react';
import { shallow } from 'enzyme';
import RightArrow from '../src/components/carousel/right-arrow';

describe('Right Arrow', () => {
  test('render the right arrow', () => {
    const wrapper = shallow(<RightArrow
      slide={() => { }}
      height={28}
      width={28}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});