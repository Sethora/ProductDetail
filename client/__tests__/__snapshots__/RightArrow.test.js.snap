// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Right Arrow render the right arrow 1`] = `
<styled.div
  height={28}
  id="right"
  onClick={[Function]}
  width={28}
>
  <img
    className="icons"
    src={null}
  />
</styled.div>
`;
